docs/mint.json

```diff
@@ -114,7 +114,8 @@
       "pages": [
         "guides/global-search",
         "guides/tenant-events",
-        "guides/credentials-export-domain"
+        "guides/credentials-export-domain",
+        "guides/cookie-monitoring"
```

Comment:
Can we add a card for this too in `getting-started.mdx`?

Reply:
![image](https://github.com/user-attachments/assets/d9c961df-50b7-4742-a68c-d956206e82ce)